acceptance/lib/helper.rb

```diff
@@ -203,11 +204,17 @@ def puppet_apply_as_puppet_user
     EOS
 
     user = master.puppet('master')['user']
+    on master, "groupadd -r puppet"
+    on master, "useradd -r -g puppet -d /opt/puppetlabs/agent/cache #{user} "
+    on master, "chown puppet:puppet /opt/puppetlabs/agent/cache/"
+    on master, "mkdir /opt/puppetlabs/agent/cache/reports"
+    on master, "chown puppet:puppet /opt/puppetlabs/agent/cache/reports"
+    on master, "chmod 0755 /opt/puppetlabs/agent/cache/reports"
```

Comment:
@rlinehan @haus we are expecting many/most of these to be removed when there are upstream fixes, correct?

@rlinehan would you mind just adding a comment line indicating that?  bonus points for creating a jira to track it and including the ticket number.